Directory.Build.props

```diff
@@ -43,6 +44,9 @@
   <PropertyGroup>
     <!-- System packages -->
     <SystemRuntimeVersion>4.3.0</SystemRuntimeVersion>
+    
+    <MicrosoftBuildVersion>15.3.409</MicrosoftBuildVersion>
```

Comment:
nit: It should go to Microsoft* section in props, alphabetical order ;-) Just to make if easier to locate these.